src/components/MyHomeMenu.spec.tsx

```diff
@@ -0,0 +1,14 @@
+import { mountWithProviders } from '../utils/testing';
+import MyHomeMenu from './MyHomeMenu';
+
+describe('MyHomeMenu', () => {
+
+    it('contains links to user pages', () => {
+        mountWithProviders(<MyHomeMenu/>);
+
+        cy.findByText('layout.my.tabs.feed');
+        cy.findByText('layout.my.tabs.todo');
+        cy.findByText('layout.my.tabs.orgs');
+        cy.findByText('layout.my.tabs.settings');
```

Comment:
I think these should use `contains()` if all you want to do is check that something exists (and not actually use it, e.g. click on it).